tests/test_fnp.py

```diff
@@ -37,17 +37,18 @@ def test_fnp_onedim(self, devices):
                 gpus=devices.gpus,
                 max_epochs=1000 if devices.use_cuda else 2,
                 logger=None,
-                check_val_every_n_epoch=100,
+                check_val_every_n_epoch=100 if devices.use_cuda else 1,
```

Comment:
This change will actually slow things down; this arguments means run every 100 epochs

Reply:
You can probably revert this part; I actually didn't see your review request and made changes to speed up FNP on my own...

Reply:
yes! but I wanted to have coverage for the validation. Thanks for the changes!